### PR TITLE
updatemsg

### PR DESCRIPTION
**Pull request by MMyatKhine**
This is the very last update of the messenger site. It might still be left some parts to change . Please have a review of my creating the copy of the responsive messenger site.

### CODE REVIEW

File: Responmessenger.css [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    align-items: center;
+    height: 80px;
+    position: sticky;
+    top: 0;
+    border-bottom: 1px solid lightgray;
+    background-color: #fafafa;
+    opacity: 0.9;
+}
+
+.logocontainer{
```
Comment (msquareprogramming) [inline Responmessenger.css]: would be great if you could be consistent in using name convention. 
```suggestion
.logoContainer{
```

File: Responmessenger.css [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    margin-left: 40px;
+}
+
+.meunItems:hover{
+    border-bottom-style: solid;
+    border-bottom-color: #385898;
+    
+}
+
+.mobilecontainer{
```
Comment (msquareprogramming) [inline Responmessenger.css:45]: [See this comment](https://github.com/MMyatKhine/Messenger_Site_Copy/pull/1/files#r671484445)

**Review by msquareprogramming — APPROVED** [review]
Suggestion

1. Be consistent in using naming convention such as camelCasing.
2. Use `Prettier` extension in VS Code to format your code so that if you work together with other people, everyone has the same coding style.

File: Responmessenger.css (hunk 1 above)
Comment (MMyatKhine) [reply Responmessenger.css]: yes sir I am gonna care to be name convention

File: Responmessenger.css (hunk 2 above)
Comment (MMyatKhine) [reply Responmessenger.css:45]: Sir, I don't know why I can't see this comment but I think it might be about to charge into  camelCasing style

**Review by MMyatKhine — COMMENTED** [review]
Ok Sir I am gonna change what you reviewed on my code.
And I had already installed that extension in VS Code but it doesn't work well.

**Pull request merged**